api/document/tests/import_xml_doc_test.py

```diff
@@ -15,3 +19,30 @@ def test_fetch_policy_pk():
 def test_fetch_policy_number():
     policy = mommy.make(Policy, omb_policy_id='M-12-13')
     assert import_xml_doc.fetch_policy('M-12-13') == policy
+
+
+@pytest.mark.django_db
+def test_import_xml_doc():
```

Comment:
This is really just an old test from `xml_importer`, but I made it run twice just to make sure the code is OK with both creating a new `DocNode` tree where none existed before, _and_ with overwriting an existing `DocNode` tree if it already exists.